bash/bldcbllib

```diff
@@ -1,29 +1,18 @@
 #!/bin/bash
 
 # bldcbllib: rebuild zcobol/lib/ZC390LIB.390
-#
-# Note: must be run from bash subdirectory
 
-# extract longest substring that ends with "/"
-dir=${0%/*}
+set -e              # exit on all errors
+cd $(dirname $0)    # set to directory of script
+cd ..               # go to root
 
-# get the z390 directory
-zdir=$(dirname $0)
-zdir=$(cd $zdir && pwd)
-zdir=$(dirname $zdir)
 
-cd ..
-
-sysmac='sysmac('$zdir'/mac)'
-syscpy='syscpy('$zdir'/mac+'$zdir'/zcobol/lib)'
-sys390='sys390('$zdir'/zcobol/lib)'
-
-${dir}/bash/asm zcobol/lib/ABORT $sysmac $syscpy $1 $2 $3 $4 $5 $6 $7 $8 $9
-${dir}/bash/asm zcobol/lib/ACCEPT $sysmac $syscpy $1 $2 $3 $4 $5 $6 $7 $8 $9
-${dir}/bash/asm zcobol/lib/CVTTOHEX $sysmac $syscpy $1 $2 $3 $4 $5 $6 $7 $8 $9
-${dir}/bash/asm zcobol/lib/DISPLAY $sysmac $syscpy $1 $2 $3 $4 $5 $6 $7 $8 $9
-${dir}/bash/asm zcobol/lib/INSPECT $sysmac $syscpy $1 $2 $3 $4 $5 $6 $7 $8 $9
-${dir}/bash/asm zcobol/lib/ZC390NUC $sysmac $syscpy $1 $2 $3 $4 $5 $6 $7 $8 $9
-${dir}/bash/asml zcobol/lib/ZC390LIB $sysmac $syscpy $sys390 RMODE24 $1 $2 $3 $4 $5 $6 $7 $8 $9
+bash/asm zcobol/lib/ABORT $sysmac $syscpy $1 $2 $3 $4 $5 $6 $7 $8 $9
+bash/asm zcobol/lib/ACCEPT $sysmac $syscpy $1 $2 $3 $4 $5 $6 $7 $8 $9
+bash/asm zcobol/lib/CVTTOHEX $sysmac $syscpy $1 $2 $3 $4 $5 $6 $7 $8 $9
+bash/asm zcobol/lib/DISPLAY $sysmac $syscpy $1 $2 $3 $4 $5 $6 $7 $8 $9
+bash/asm zcobol/lib/INSPECT $sysmac $syscpy $1 $2 $3 $4 $5 $6 $7 $8 $9
+bash/asm zcobol/lib/ZC390NUC $sysmac $syscpy $1 $2 $3 $4 $5 $6 $7 $8 $9
+bash/asml zcobol/lib/ZC390LIB $sysmac $syscpy $sys390 RMODE24 $1 $2 $3 $4 $5 $6 $7 $8 $9
```

Comment:
Since you deleted the setting of sysmac, syscpy, and sys390, aren't they null in the replaced lines? Is this intentional?

Reply:
That is interesting.. I probably did that accidentally, but it still works.
The asm and asml scripts already do a sysmac and syscpy include, but it does not zcobol/lib include.
Maybe this job does not need these includes/macs/libs...